lisa/tests/staging/sched_android.py

```diff
@@ -0,0 +1,290 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2019, Arm Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import os.path
+import abc
+
+from lisa.wlgen.rta import RTA, Periodic
+from lisa.tests.base import ResultBundle, CannotCreateError, RTATestBundle
+from lisa.trace import Trace, FtraceCollector, FtraceConf, requires_events
+from lisa.target import Target
+from lisa.utils import ArtifactPath
+
+class SchedTuneTest(RTATestBundle, abc.ABC):
+    """
+    Abstract class for SchedTune testing
+
+    :param boost: The SchedTune `boost` level to apply the rtapp's cgroup
```

Comment:
Those aren't parameters but class attributes (otherwise they'd show up in `__init__()`), so you should document them as such. Here's an example: https://github.com/ARM-software/lisa/blob/next/lisa/tests/base.py#L242

Also it looks like you use them as class constants, so they should be uppercase.